overlord/snapstate/autorefresh.go

```diff
@@ -126,6 +127,27 @@ func (m *autoRefresh) clearRefreshHold() {
 	tr.Commit()
 }
 
+// AtSeed configures refresh policies at end of seeding.
+func (m *autoRefresh) AtSeed() error {
+	// on classic hold refreshes for 2h after seeding
+	if release.OnClassic {
+		var t1 time.Time
+		tr := config.NewTransaction(m.state)
+		err := tr.Get("core", "refresh.hold", &t1)
+		if !config.IsNoOption(err) {
+			// already set or error
+			return err
+		}
+		// TODO: have a policy that if the snapd exe itself
+		// is older than X weeks/months we skip the holding?
+		now := time.Now().UTC()
+		tr.Set("core", "refresh.hold", now.Add(2*time.Hour))
```

Comment:
Silly question: The PR title talks about 6h (and iirc we discussed 6h in the past). What is the background about changing it to 2h now?

Reply:
it's the result of various back and forth, with the default schedule we will still pick something in a 6h window anyhow, but we hold for 2h now